crates/modelardb_compression/src/models/mod.rs

```diff
@@ -442,11 +512,17 @@ mod tests {
     }
 
     // Tests for compress_residual_value_range().
+    // TODO
```

Comment:
If this `TODO` should be kept maybe add some text to make it clear what should be done.

Reply:
Deleted as it was a reminder that I should write tests that I forgot to delete.